src/sentry/api/event_search.py

```diff
@@ -584,6 +585,21 @@ def convert_search_filter_to_snuba_query(search_filter):
                 )
             )
         return [name, search_filter.operator, internal_value]
+    elif name == "trace":
+        if not search_filter.value.raw_value:
+            operator = "IS NULL" if search_filter.operator == "=" else "IS NOT NULL"
+            return [name, operator, None]
+
+        try:
+            return [
+                name,
+                search_filter.operator,
+                six.binary_type(uuid.UUID(search_filter.value.raw_value)),
```

Comment:
Why do you want bytes here? This will not work out in python3 as bytes can't be serialized to JSON.

Reply:
My bad. That's what flake recommended so I just used that. But you're right.